packages/web-components/src/utils/utils.ts

```diff
@@ -37,3 +37,37 @@ export function consoleDevError(message: string): void {
     console.error(message);
   }
 }
+
+/**
+ *
+ * Take the length of a string; if the length is "1" then return blank string for no plurality
+ * otherwise return an 's' for plurality
+ */
+export function plurality(length: number): string {
+  return length === 1 ? '' : 's';
+}
+
+/**
+ * generates message to update character count if
+ * currently used in va-text-input and va-text-area v3 components
+ * update this method with translations instead of string literals
```

Comment:
> update this method with translations instead of string literals

Just double checking; was the translations spreadsheet updated with any new text we need to translate?

Reply:
> > update this method with translations instead of string literals
> 
> Just double checking; was the translations spreadsheet updated with any new text we need to translate?

Good call - doing now